pkg/util/patch/patch.go

```diff
@@ -56,20 +56,20 @@ func (p Patch) Clear() {
 	}
 }
 
-func (p Patch) MustTranslate(path string, translate func(path string, val interface{}) (interface{}, error)) {
+func (p Patch) MustTranslate(path string, translate func(path string, val interface{}, exists bool) (interface{}, error)) {
```

Comment:
nit: could we define a type for the `translate` func to avoid copying the signature in multiple places?